package.json

```diff
@@ -11,7 +11,7 @@
     "test": "npm run build && npm run test:fmt && npm run test:ts && npm run test:unit",
     "test:fmt": "prettier src/*.js test/*.js *.*d.ts extend-expect.js --check",
     "test:ts": "tsd",
-    "test:unit": "npx jest@^${JEST_VERSION:-25}"
+    "test:unit": "npx jest@^${JEST_VERSION:-27}"
```

Comment:
Looking at this now, I think I set this up this way so I could test multiple Jest versions. Separate PR, but could you expand the matrix to include testing multiple Jest versions?

Reply:
Yeah, sorry I considered that when I did this change but I forgot to work on this.

Reply:
Done in https://github.com/kefirjs/jest-kefir/pull/407